docs/topics/Gateway_Events.md

```diff
@@ -904,14 +904,15 @@ Sent when a user adds a reaction to a message.
 
-| Field      | Type                                                         | Description                                                                                |
-| ---------- | ------------------------------------------------------------ | ------------------------------------------------------------------------------------------ |
-| user_id    | snowflake                                                    | ID of the user                                                                             |
-| channel_id | snowflake                                                    | ID of the channel                                                                          |
-| message_id | snowflake                                                    | ID of the message                                                                          |
-| guild_id?  | snowflake                                                    | ID of the guild                                                                            |
-| member?    | [member](#DOCS_RESOURCES_GUILD/guild-member-object) object   | Member who reacted if this happened in a guild                                             |
-| emoji      | a partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | Emoji used to react - [example](#DOCS_RESOURCES_EMOJI/emoji-object-standard-emoji-example) |
+| Field             | Type                                                         | Description                                                                                |
+| ----------------- | ------------------------------------------------------------ | ------------------------------------------------------------------------------------------ |
+| user_id           | snowflake                                                    | ID of the user                                                                             |
+| channel_id        | snowflake                                                    | ID of the channel                                                                          |
+| message_id        | snowflake                                                    | ID of the message                                                                          |
+| guild_id?         | snowflake                                                    | ID of the guild                                                                            |
+| member?           | [member](#DOCS_RESOURCES_GUILD/guild-member-object) object   | Member who reacted if this happened in a guild                                             |
+| emoji             | a partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | Emoji used to react - [example](#DOCS_RESOURCES_EMOJI/emoji-object-standard-emoji-example) |
+| message_author_id | snowflake                                                    | ID of the user that authored the message that was reacted to                                 |
```

Comment:
```suggestion
| message_author_id | snowflake                                                    | ID of the user who authored the message that was reacted to                                 |
```
I'd say, that way the description sounds better...

Reply:
I think both "that" and "who" are clear enough so I don't care which one to use, but a bot could also be a message author, and "who" isn't exactly correct for bots AFAIK.

Reply:
I think it's more about having 2 "that" in the same sentence. correct but maybe a bit weird?

Reply:
ok I guess who sounds better